packages/completer/src/handler.ts

```diff
@@ -477,6 +511,93 @@ export namespace CompletionHandler {
     connector: IDataConnector<IReply, void, IRequest>;
   }
 
+  /**
+   * Wrapper object for ICompletionItem.
+   */
+  export interface ICompletionItems {
+    /**
+     * isIncomplete is true if this is a partial list of completion items.
+     */
+    isIncomplete: boolean;
```

Comment:
What is the use case for this? I do not see `isIncomplete` being accesses anywhere, nor being set to true in any of tests.

Reply:
Ok, I see this discussed in https://github.com/jupyterlab/jupyterlab/issues/7044#issuecomment-594249225 - sorry. Maybe larger parts of the comment could be embedded in the interface documentation?

Reply:
I've opted to remove `isIncomplete` for now. On second thought, this is probably best tackled in a follow up issue / PR.